spec/acts_as_taggable_on/dirty_spec.rb

```diff
@@ -123,5 +119,19 @@
         expect(@taggable.changes).to be_empty
       end
     end
+
+    context 'when language_list changed by association' do
+      let(:tag) { ActsAsTaggableOn::Tag.new(name: 'one') }
+
+      before(:each) do
```

Comment:
I would refactor this to 
```ruby
    it 'flags language_list as changed' do
        expect(@taggable.changes).to be_empty
        @taggable.languages << tag
        expect(@taggable.language_list_changed?).to be_truthy
end
```

Unless there is a specific guideline around it, I usually tend to not over-use the before-each, harder to read. Since here you have only one test in your context, I would drop it.